src/Runner.Common/Unit.cs

```diff
@@ -0,0 +1,8 @@
+// Represents absence of value.
+namespace GitHub.Runner.Common
+{
+    public readonly struct Unit
```

Comment:
do we have to have this struct?

Reply:
It's used in `RunnerService`, to have a way to get data "empty" data for Task. I think it's nice to have this, can be used for any later uses. [Dotnet code has it as well](https://github.com/github/actions-dotnet/blob/b5850133b3c04d4293f029eb911e7fdcb3bd2987/Launch/Sdk/Server/Unit.cs)